db/migrate/20170109115157_fix_lookup_key_auditable_type.rb

```diff
@@ -0,0 +1,12 @@
+class FixLookupKeyAuditableType < ActiveRecord::Migration
+  def up
+    Audit.where(:auditable_type => 'LookupKey').each do |audit|
```

Comment:
This would potentially load a very large amount of audits from the DB to check their types. would it be possible to perform this in the database without loading all of the audits and going over them one by one? (maybe change all the puppetLKs in one query and all the variableLKs in a second?) If it isn't possible, please use `find_each` instead of `each` so at least this is done in batches.